### PR TITLE
[DO NOT MERGE] forge fmt demo

### PR DESCRIPTION
**Pull request by RensR**
Try out `forge fmt`

### CODE REVIEW

File: contracts/src/v0.8/ccip/ocr/OCR2Base.sol [hunk 1]
```diff
@@ -68,17 +69,15 @@ abstract contract OCR2Base is OwnerIsCreator, OCR2Abstract {
...
-      3 + // 3 words containing reportContext
-      32 + // word containing start location of abiencoded report value
-      32 + // word containing location start of abiencoded rs value
-      32 + // word containing start location of abiencoded ss value
-      32 + // rawVs value
-      32 + // word containing length of report
-      32 + // word containing length rs
-      32; // word containing length of ss
+  uint16 private constant TRANSMIT_MSGDATA_CONSTANT_LENGTH_COMPONENT = 4 // function selector
+    + 32 * 3 // 3 words containing reportContext
```
Comment (RensR) [inline contracts/src/v0.8/ccip/ocr/OCR2Base.sol:73]: Love this, makes so much more sense

File: contracts/src/v0.8/ccip/ARM.sol [hunk 2]
```diff
@@ -419,9 +412,11 @@ contract ARM is IARM, OwnerIsCreator, ITypeAndVersion {
   /// @return accumulatedWeight sum of weights of voters, will be zero if voting took place with an older config version
   /// @return blessed will be accurate regardless of when voting took place
   /// @dev This is a helper method for offchain code so efficiency is not really a concern.
-  function getBlessProgress(
-    IARM.TaggedRoot calldata taggedRoot
-  ) external view returns (address[] memory blessVoteAddrs, uint16 accumulatedWeight, bool blessed) {
+  function getBlessProgress(IARM.TaggedRoot calldata taggedRoot)
+    external
+    view
+    returns (address[] memory blessVoteAddrs, uint16 accumulatedWeight, bool blessed)
```
Comment (RensR) [inline contracts/src/v0.8/ccip/ARM.sol:418]: kinda like this

File: contracts/src/v0.8/ccip/ARM.sol [hunk 3]
```diff
@@ -480,10 +475,8 @@ contract ARM is IARM, OwnerIsCreator, ITypeAndVersion {
 
   function _validateConfig(Config memory config) internal pure returns (bool) {
     if (
-      config.voters.length == 0 ||
-      config.voters.length > MAX_NUM_VOTERS ||
-      config.blessWeightThreshold == 0 ||
-      config.curseWeightThreshold == 0
+      config.voters.length == 0 || config.voters.length > MAX_NUM_VOTERS || config.blessWeightThreshold == 0
```
Comment (RensR) [inline contracts/src/v0.8/ccip/ARM.sol:478]: don't like this

File: contracts/src/v0.8/ccip/ARM.sol [hunk 4]
```diff
@@ -327,9 +321,8 @@ contract ARM is IARM, OwnerIsCreator, ITypeAndVersion {
       curserRecord.cursesHash,
       curseVoteProgress.accumulatedWeight
     );
-    if (
-      !curseVoteProgress.curseActive && curseVoteProgress.accumulatedWeight >= curseVoteProgress.curseWeightThreshold
-    ) {
+    if (!curseVoteProgress.curseActive && curseVoteProgress.accumulatedWeight >= curseVoteProgress.curseWeightThreshold)
+    {
```
Comment (matYang) [inline contracts/src/v0.8/ccip/ARM.sol:325]: not liking this bracket being punted to new line v.s. rearranging the if conditions

**Pull request closed without merging**